src/frontend/screens/Settings/Settings.js

```diff
@@ -99,6 +99,11 @@ const m = defineMessages({
     defaultMessage: "App Passcode and Device Security",
     description: "Description of security button in settings",
   },
+  mapSettings: {
+    id: "screens.Settings.mapSettings",
+    defaultMessage: "Map Settings",
+    description: "Description of map settings button in settings",
```

Comment:
nit but this should highlight that it's the option's title (as opposed to the subtitle/description)